app/controllers/api/move_events_controller.rb

```diff
@@ -4,81 +4,72 @@ module Api
   class MoveEventsController < ApiController
     include Moves::Eventable
 
-    ACCEPT_PARAMS = [:type, attributes: %i[timestamp notes]].freeze
+    APPROVE_PARAMS = [:type, attributes: %i[timestamp date]].freeze
     CANCEL_PARAMS = [:type, attributes: %i[timestamp cancellation_reason cancellation_reason_comment notes]].freeze
-    COMPLETE_PARAMS = [:type, attributes: %i[timestamp notes]].freeze
     LOCKOUT_PARAMS = [:type, attributes: %i[timestamp notes], relationships: { from_location: {} }].freeze
     REDIRECT_PARAMS = [:type, attributes: %i[timestamp notes], relationships: { to_location: {} }].freeze
-    APPROVE_PARAMS = [:type, attributes: %i[timestamp date]].freeze
     REJECT_PARAMS = [:type, attributes: %i[timestamp rejection_reason cancellation_reason_comment rebook]].freeze
 
+    STANDARD_PARAMS = [:type, attributes: %i[timestamp notes]].freeze # for accept, complete and start move events
```

Comment:
What about calling this `COMMON_PARAMS` or `SHARED_PARAMS`?

Reply:
`COMMON_PARAMS` sounds good 👍  I'll rename this